runtimes/aleph-alpine-3.13-python/loading.html

```diff
@@ -0,0 +1,443 @@
+        }
+
+        25% {
+            background: #a0d2eb;
+        }
+
+        50% {
+            background: #ffa8b6;
+        }
+
+        75% {
+            background: #d0bdf4;
+        }
+
+        100% {
+            background: #494d5f;
+        }
+    }
+
+    @keyframes upndown {
+        0% {
+            transform: translateY(5px);
+        }
+        50% {
+            transform: translateY(15px);
+        }
+        100% {
+            transform: translateY(5px);
+        }
+    }
+    @keyframes smallnbig {
+        0% {
+            width: 90px;
+        }
+        50% {
+            width: 100px;
+        }
+        100% {
+            width: 90px;
+        }
+    }
+    @keyframes shine {
+        0% {
+            opacity: 0.2;
+        }
+        25% {
+            opacity: 0.1;
+        }
+        50% {
+            opacity: 0.2;
+        }
+        100% {
+            opacity: 0.2;
+        }
+    }
+</style>
+<script type="text/javascript">
+    const pageX = window.innerWidth;
+    const pageY = window.innerHeight;
+    let mouseY = 0;
+    let mouseX = 0;
+
+    window.addEventListener("mousemove", function( event ) {
+        //verticalAxis
+        mouseY = event.clientY;
+        const yAxis = -((pageY / 2 - mouseY) / pageY * 300);
+        //horizontalAxis
+        mouseX = event.clientX / - pageX;
+        const xAxis = - mouseX * 100 - 100;
+
+        document.getElementsByClassName('box__ghost-eyes')[0].setAttribute('style', 'transform: translate('+ xAxis +'%,'+ yAxis +'%)');
+    });
+
+    setTimeout(function() {
+        location.reload();
+    }, 5000);
+</script>
+<div class="box">
+    <div class="box__description">
+        <div class="box__ghost">
+            <div class="symbol"></div>
+            <div class="symbol"></div>
+            <div class="symbol"></div>
+            <div class="symbol"></div>
+            <div class="symbol"></div>
+            <div class="symbol"></div>
+
+            <div class="box__ghost-container">
+                <div class="box__ghost-eyes">
+                    <div class="box__eye-left"></div>
+                    <div class="box__eye-right"></div>
+                </div>
+                <div class="box__ghost-bottom">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                </div>
+            </div>
+            <div class="box__ghost-shadow"></div>
+        </div>
+        <div class="loading__container">
+            <div class="loading__box"></div>
+        </div>
+        <div class="box__description-container">
+            <div class="box__description-title">Whoops!</div>
+            <div class="box__description-text">Seems like your VM is still loading, please wait...</div>
+        </div>
+
+        <a href="#" onclick="location.reload()" target="_self" class="box__button">Refresh!</a>
+    </div>
+
+</div>
+</body>
+</html>
```

Comment:
Would it make sense to load the JS and CSS from IPFS instead of bundling it into the runtime ?

Reply:
If we load it from IPFS will be so complicated to give the users the possibility to customize their loading pages by the runtime.